VideoLocker/res/layout/fragment_course_unit_webview.xml

```diff
@@ -15,4 +15,8 @@
         android:layout_gravity="center_vertical|center_horizontal"
         android:visibility="gone" />
 
+    <include
+        layout="@layout/no_connectivity_layout"
+        android:visibility="gone" />
```

Comment:
`android:visibility="gone"` should probably be defined in the layout itself, as it should always be hidden at the start.